server/routes/search.ts

```diff
@@ -52,8 +52,8 @@ export default function routes({ auditService, prisonerSearchService, visitSessi
   get(['/prisoner/results', '/prisoner-visit/results'], async (req, res) => {
     const { prisonId } = req.session.selectedEstablishment
     const isVisit = req.originalUrl.includes('-visit')
-    const search = (req.query.search || '') as string
-    const currentPage = (req.query.page || '') as string
+    const search = typeof req.query.search === 'string' ? req.query.search : ''
```

Comment:
CodeQL flagged these as unsafe because query params can be of type `string` or `string[]`. Just casting it with `as string` placates typescript but doesn't address the actual problem of inadvertently letting an array through when you're just expecting a string.